_includes/sidebar.html

```diff
@@ -3,10 +3,12 @@
   <li class="sidebar-title">Get started</li>
 
     <ul class="sidebar-items">
-      <li><a href="/setup/">Install</a></li>
-      <li><a href="/getting-started/">Create and run your first app</a></li>
-      <li><a href="/ide-setup/">Setting up the IDE</a></li>
-      <li><a href="/bootstrap-into-dart/">Bootstrap into Dart</a></li>
+      <li><a href="/get-started/">1: Overview</a></li>
+      <li><a href="/get-started/install/">2: Install Flutter</a></li>
+      <li><a href="/get-started/editor-configuration/">3: Editor configuration</a></li>
+      <li><a href="/get-started/test-drive/">4: Test drive</a></li>
+      <li><a href="/get-started/codelab/">5: Core concepts codelab</a></li>
+      <li><a href="/get-started/learn-more/">6: Learn more</a></li>
```

Comment:
I think we should lose this from the sidenav. Only link it from the end of the codelab.

For independent "learn more", we'll have flutter.io/docs

Reply:
I'm going to keep this for now, havn't fully understood how /docs plays into this.